README.md

```diff
@@ -226,6 +226,22 @@ The hash for StremioServiceSetup.exe is <exe sha256 hash>
 
 If the `--quiet` flag is used together with `--dry-run` only the descriptor is printed to `STDOUT`. In case of error the error is printed to `STDERR` and `STDOUT` is blank.
 
+## Development
+
+
+### Updating the copyright text in files
```

Comment:
I don't think this is needed, we are the ones updating the copyright headers, this doc suggest that contributors should do it
Also it could be done just by opening a code editor and do a project string replace

Reply:
yes, it can, but I've added this documentation so that we can fully automate this with CI in the future and didn't want my findings to disappear but also help me with the next year update ;)

Reply:
I've removed it and added it to the #67 PR